src/components/alert/IconAlert.vue

```diff
@@ -1,9 +1,9 @@
 <template>
     <el-alert
-    :title="title"
-    :type="type"
-    show-icon>
-  </el-alert>
+      :title="title"
```

Comment:
Indentation can be updated. We can introduce some eslint configuration for more strictly maintained code... what do you think?

Reply:
Yes, please. I need to install some kind to help with this issue. Let me know if you know of any? Also yes if we can add it in our tests too

Reply:
I worked in that environment but didn't configured earlier. Ok you can add a ticket. I will look into...